admin-core/src/main/resources/application.conf

```diff
@@ -74,7 +74,13 @@ drill {
         lifetime = "1h"
         lifetime = ${?DRILL_JWT_LIFETIME}
     }
+    passwordRequirements {
+        minLength = 6
+        mustHaveUppercase = false
+        mustHaveLowercase = false
+        mustHaveDigit = false
+    }
     users = [
-        "{\"username\": \"guest\", \"password\": \"guest\", \"role\": \"ADMIN\"}"
+        "{\"username\": \"guest\", \"password\": \"guest\", \"role\": \"USER\"}"
```

Comment:
_tbd_ - I don't get why we call it "guest" user if it has "USER" role with basically all permissions to read and edit AUT-related data